doc/builders/special/runinmkshell.section.md

```diff
@@ -0,0 +1,64 @@
+# pkgs.runInMkShell {#sec-pkgs-runInMkShell}
+
+`pkgs.runInMkShell` is a function that generates a wrapper script that
+receives the command to be run as arguments and then run it inside a special
+environment created with [`mkShell`](#sec-pkgs-mkShell).
```

Comment:
Best to describe in terms of what it does. Something about a wrapper script that provides an environment similar to what Nix and stdenv provide.